File: Conditionals/time.py
# Lizzie DeLong, How to get time
import time

#first instance of time in programing
first_time = time.gmtime()
# print(first_time)

#current time in seconds since January 1st, 1970
current = time.time()
# print(current)

#current sate and time like we see it normally
now  = time.ctime(current)
# print(now)

#getting a part of the time
localTime = time.localtime(current)
day = localTime.tm_wday
hour = localTime.tm_hour # In military time (0-23)
print(hour)

File: Conditionals/time.c
// Lizzie DeLong, Getting the time
#include <stdio.h>
#include <time.h>

int main(void){
    // Time since Jan 1 1970
    time_t seconds;
    seconds = time(NULL);

    // printf("Seconds since Jan 1, 1970 = %d\n", seconds);

    // Get current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    //printf("Current time and date is %s\n", asctime(timeinfo));

    //current hour
    time_t now = time(NULL);

    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour; // Military Tine

    printf("%d\n", hour);

    return 0;
}